doc/release/README.compiling.rst

```diff
@@ -0,0 +1,77 @@
+=========================
+Compiling Chapel Programs
+=========================
+
+The Chapel compiler converts programs expressed in Chapel source code
+into the corresponding executable form.  This document briefly discusses
+running the Chapel compiler.
+
+.. contents::
+
+
+------------
+Getting Help
+------------
+
+If you have set up your environment as described in
+``$CHPL_HOME/doc/README`` and built the compiler as described in
+``$CHPL_HOME/doc/README.building``, you can see the Chapel compiler's
```

Comment:
We will have to come back and add direct cross references at some point in the near future.